types/src/verifier/mod.rs

```diff
@@ -84,9 +85,10 @@ fn to_fil_public_replica_infos(
     let replicas = src
         .iter()
         .map::<Result<(SectorId, PublicReplicaInfo), String>, _>(|sector_info: &SectorInfo| {
+            let proof_v2: RegisteredSealProofV2 = RegisteredSealProof(sector_info.proof).into();
```

Comment:
Could we use the new proof? How much did those change between FVM2 and FVM3?

Reply:
Ok, looks like i'll also need to move `RegisteredPoStProof` as:

```rust
            let proof_v2: RegisteredSealProofV2 = RegisteredSealProof::from(sector_info.proof).into();
            let commr = cid_to_replica_commitment_v1(&sector_info.sealed_cid)?;
            let proof = match typ {
                ProofType::Winning => proof_v2.registered_winning_post_proof()?,
                // ProofType::Window => sector_info.proof.registered_window_post_proof()?,
            };
```

`registered_winning_post_proof` needs to come from fvm3 for me to use the v3 shim

Reply:
no additional shims were needed. updated to use v3 proof shim.